.azure_pipelines/job_templates/olive-setup-template.yaml

```diff
@@ -25,3 +25,6 @@ steps:
   - script: |
       pip install ${{ parameters.onnxruntime }}
     displayName: Install ${{ parameters.onnxruntime }}
+
+- script: pip install onnxscript
+  displayName: Install onnxscript
```

Comment:
since torch export is key olive feature and onnxscript is used in the new torch versions for it, I think we should just add it as a required dependency for olive once it has a stable release? 
For the time being, let's add it to https://github.com/microsoft/Olive/blob/main/test/requirements-test.txt instead?